src/redux/reducers/workspaces-reducer.ts

```diff
@@ -44,6 +44,10 @@ const setWorkspace = (state: WorkspacesStore, { payload }: { payload: { data: Wo
   isLoading: false,
 });
 
+export const isWorkspace = (data: unknown | Workspace): data is Workspace => {
```

Comment:
@Hyperkid123 Great suggestion. This is definitely cleaner and allowed me to delete the casting in each validator for `id`